giraffe/README.md

```diff
@@ -307,12 +307,16 @@ When using the comma separated values (CSV) from the Flux query as the `fluxResp
   - "linear" scaling means the same distance between ticks represents the same increase in value.
   - "log" (logarithmic) scaling means the same distance between ticks can represent an exponential increase in value, used as a way to display data with a very wide range of values in a compact space.
 
+- :warning: When using `xDomain`, `onSetXDomain` or `onResetXDomain` all three must be set or neither of them will work
+
```

Comment:
Does not look like an effective warning. This is what I see in a markdown viewer:
<img width="1084" alt="Screen Shot 2020-10-26 at 3 05 29 PM" src="https://user-images.githubusercontent.com/10736577/97233796-c7d69d00-179c-11eb-99b5-27b65c7aaece.png">